frontend-react/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "0.1.1",
     "private": true,
     "npmClient": "yarn",
-    "proxy": "http://localhost:7071",
+    "proxy": "https://staging.prime.cdc.gov",
```

Comment:
can we not check this line in so devs don't get confused where their back-end is pointing? (haha that sounds funny :smile: )

Reply:
AHH! My bad, totally forgot I'd changed this and did a `git add .` -- I'll fix this up